src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -825,7 +825,7 @@ public function handleBookEdit(BookEditPacket $packet) : bool{
 			case BookEditPacket::TYPE_SWAP_PAGES:
 				if(!$newBook->pageExists($packet->pageNumber) || !$newBook->pageExists($packet->secondaryPageNumber)){
 					//the client will create pages on its own without telling us until it tries to switch them
-					$newBook->addPage(max($packet->pageNumber, $packet->secondaryPageNumber));
+					$newBook->setPageText(max($packet->pageNumber, $packet->secondaryPageNumber));
```

Comment:
This is no longer idiomatic. It's not obvious what this code does anymore.

Reply:
I did not see that this was already checked in setPageText. 
Unnecessary repetition, might as well delete this part of the code